docs/pages/ui-programming/react-native-gradients.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Linear gradients in Expo and React Native apps
+sidebar_title: Using linear gradients
+---
+
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+import TerminalBlock from '~/components/plugins/TerminalBlock';
+import SnackInline from '~/components/plugins/SnackInline';
+
+To add linear gradients to our Expo and React Native apps, we're going to use `expo-linear-gradient`. We're using this package because it is included in the Expo SDK and also because gradients are not otherwise supported out of the box with React Native.
```

Comment:
```suggestion
This page will demonstrate how to style components with linear gradients using the `expo-linear-gradient`. This package provides a simple API and comes pre-installed in the Expo Go and will work in any React Native project, however you are choose to build it.
```

Reply:
A screenshot of the component you've created so folks can immediately see what is being made and confirm its what they're looking for would be great.